xp/xconnect/DockerFile

```diff
@@ -26,11 +26,11 @@ ARG XCONNECT_PACKAGE
 
 SHELL ["powershell", "-NoProfile", "-Command", "$ErrorActionPreference = 'Stop';"] 
 
-COPY files/license.xml /Files/
 COPY files/$XCONNECT_PACKAGE /Files/
 COPY files/*.pfx /Files/
 COPY --from=prepare /Files/Config /Files/Config/
 COPY xp/xconnect/disable-solr-ssl.json /Files/Config/
+COPY xp/xconnect/xconnect-xp0.json /Files/Config/
```

Comment:
deze heb je al toch? (in $Env:SIF_CONFIG)

Reply:
Nee ik moest de json iets aanpassen. Is nu niet meer relevant omdat die opzet niet werkte.